tests/unit/mainwindow/statusbar/test_url.py

```diff
@@ -0,0 +1,126 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2014-2016 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+
+"""Test Statusbar url."""
+
+import pytest
+
+from collections import namedtuple
+from qutebrowser.mainwindow.statusbar.url import UrlText
```

Comment:
This is probably a pet peeve of mine, but I really don't like `from module import thing` imports - `from package import module` is okay, but please use `collections.namedtuple` and `url.UrlText` in the code and adjust the imports accordingly.